diff --git a/node_modules/@apollosproject/ui-mapview/src/MapView/index.js b/node_modules/@apollosproject/ui-mapview/src/MapView/index.js
index c6f91dd..268be92 100644
--- a/node_modules/@apollosproject/ui-mapview/src/MapView/index.js
+++ b/node_modules/@apollosproject/ui-mapview/src/MapView/index.js
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import RNMapView from 'react-native-maps';
+// import RNMapView from 'react-native-maps';
 import { Animated, Dimensions, Platform, PixelRatio } from 'react-native';
 import { SafeAreaView } from 'react-navigation';
 import { debounce, isNil } from 'lodash';
diff --git a/node_modules/@apollosproject/ui-mapview/src/Marker/index.js b/node_modules/@apollosproject/ui-mapview/src/Marker/index.js
index e5e0780..9048943 100644
--- a/node_modules/@apollosproject/ui-mapview/src/Marker/index.js
+++ b/node_modules/@apollosproject/ui-mapview/src/Marker/index.js
@@ -1,7 +1,7 @@
 import React, { memo } from 'react';
 import { View, Animated } from 'react-native';
 import PropTypes from 'prop-types';
-import { Marker as DefaultMarker } from 'react-native-maps';
+// import { Marker as DefaultMarker } from 'react-native-maps';
 import Color from 'color';
 
 import styled from '@apollosproject/ui-kit/src/styled';
