diff --git a/node_modules/@apollosproject/ui-notifications/src/Provider.js b/node_modules/@apollosproject/ui-notifications/src/Provider.js
index e191456..834d9e7 100644
--- a/node_modules/@apollosproject/ui-notifications/src/Provider.js
+++ b/node_modules/@apollosproject/ui-notifications/src/Provider.js
@@ -3,8 +3,8 @@ import querystring from 'querystring';
 import React, { Component } from 'react';
 import { Linking } from 'react-native';
 import PropTypes from 'prop-types';
-import { gql } from '@apollo/client';
-import { withApollo } from '@apollo/client/react/hoc';
+import gql from 'graphql-tag';
+import { withApollo } from 'react-apollo';
 import { get } from 'lodash';
 import OneSignal from 'react-native-onesignal';
 import ApollosConfig from '@apollosproject/config';
diff --git a/node_modules/@apollosproject/ui-notifications/src/permissionUtils.js b/node_modules/@apollosproject/ui-notifications/src/permissionUtils.js
index b382f4c..9c47fc5 100644
--- a/node_modules/@apollosproject/ui-notifications/src/permissionUtils.js
+++ b/node_modules/@apollosproject/ui-notifications/src/permissionUtils.js
@@ -1,7 +1,7 @@
 /* eslint-disable no-empty */
 
 import OneSignal from 'react-native-onesignal';
-import { gql } from '@apollo/client';
+import gql from 'graphql-tag';
 import {
   checkNotifications,
   openSettings,
diff --git a/node_modules/@apollosproject/ui-notifications/src/updatePushId.js b/node_modules/@apollosproject/ui-notifications/src/updatePushId.js
index cc86eee..a43b658 100644
--- a/node_modules/@apollosproject/ui-notifications/src/updatePushId.js
+++ b/node_modules/@apollosproject/ui-notifications/src/updatePushId.js
@@ -1,4 +1,4 @@
-import { gql } from '@apollo/client';
+import gql from 'graphql-tag';
 
 const UPDATE_PUSH_ID = gql`
   mutation updateUserPushSettings($input: PushSettingsInput!) {
